e2e/pages/adf/searchFiltersPage.ts

```diff
@@ -108,9 +133,8 @@ export class SearchFiltersPage {
 
         this.showMoreButtonForSize.isDisplayed().then(async (visible) => {
             if (visible) {
+                let totalNumberOfCheckboxes = await this.numberOfCheckboxesforSize.count();
```

Comment:
Is clickSizeShowMoreButtonUntilIsNotDisplayed() method used? I don't get the scope of this method, especially after line 136 was moved out of for statement.